libs/vkd3d/raytracing_pipeline.c

```diff
@@ -110,14 +137,7 @@ static ULONG d3d12_state_object_release(struct d3d12_state_object *state_object)
     TRACE("%p decreasing refcount to %u.\n", state_object, refcount);
 
     if (!refcount)
-    {
```

Comment:
Can there ever be a situation where the app can re-claim a reference to the object after the (public) ref count has reached zero? In that case, `AddRef` would have to handle that case and add a private ref again.

Reply:
I doubt it. The DXR wording says: "Collection to include in state object. Enclosing state object holds a ref on the existing collection."
I proved it's not a real reference, but I don't think this means being able to make those objects live again.